src/Mono.Android/Mono.Android.csproj

```diff
@@ -308,21 +312,10 @@
     <Compile Include="Javax.Microedition.Khronos.Egl\EGLContext.cs" />
     <Compile Include="Org.Apache.Http.Impl.Conn\DefaultClientConnection.cs" />
     <Compile Include="Org.Apache.Http.Impl.Cookie\BasicClientCookie.cs" />
-    <Compile Include="System.Drawing/Color.cs" />
```

Comment:
Shouldn't we specify the [`TypeForwardedToAttribute`](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.typeforwardedtoattribute?view=netframework-4.7.2) for each of these types, instead of not mentioning anything at all?

Reply:
@jonpryor The `external/mono` conflict is obscuring that the `mono/2018-10` includes these two commits on `mono/corefx`:

https://github.com/mono/corefx/commit/20afe8ab8d2166c51348fc86ddd52043a2decaae and https://github.com/mono/corefx/commit/6cb38b3f4a9704c6d194326ce187b2ff5b3c591c

Reply:
@alexischr: those commits you mentioned are `TypeForwardedFrom`.  I'm suggesting `TypeForwardedTo`.  From vs. To. Different attributes. :-)

I thus repeat: shouldn't we add the following (and more!) to `Mono.Android.dll`?

```csharp
[assembly:TypeForwardedTo(typeof(System.Drawing.Rectangle)]
[assembly:TypeForwardedTo(typeof(System.Drawing.RectangleF)]
// ...
```

Reply:
@jonpryor I don't understand type forwarders too well - I thought if you had one you didn't need the other. These are now added with https://github.com/xamarin/xamarin-android/pull/2253/commits/06c387f027f85446ca4e62bd105125c9bf7bf9c0